nautobot/utilities/fields.py

```diff
@@ -189,3 +194,19 @@ def formfield(self, **kwargs):
                 **kwargs,
             }
         )
+
+
+class TimeZoneField(TZField):
+    """Overloads time_zone.TimeZoneField to represent blank value as "" instead of null."""
+
+    def _get_python_and_db_repr(self, value):
+        "Returns a tuple of (python representation, db representation)"
+        if value is None or value == "":
+            return ("", "")
+        if is_pytz_instance(value):
+            return (value, value.zone)
+        try:
+            return (pytz.timezone(force_str(value)), force_str(value))
+        except pytz.UnknownTimeZoneError:
+            pass
+        raise ValidationError("Invalid timezone '%s'" % value)
```

Comment:
Is this pattern necessary for some reason? Or could we just:
```suggestion
            raise ValidationError("Invalid timezone '%s'" % value)
```